styles/pages/_product.scss

```diff
@@ -15,13 +15,12 @@
   &__description {
     font-weight: 300;
     line-height: 1.5;
-    margin-bottom: 2rem;
+    margin: 2rem 0;
```

Comment:
Should this just be in px?

Reply:
If we don't want the spacing to change if the user changes the text size? 🤷‍♂️

Reply:
Changed